src/css/_custom.css

```diff
@@ -335,3 +335,7 @@ body {
 .fill-screen {
   min-height: 100vh;
 }
+
+.video-height {
```

Comment:
What's the thinking behind making this a pixel-height? Sure that there is one but just curious ;)